src/components/layout/dt-alert/dt-alert.js

```diff
@@ -0,0 +1,183 @@
+        align-content: center;
+        align-items: center;
+      }
+
+      .dt-alert.dt-alert--outline {
+        background-color: transparent;
+        color: var( --dt-alert-context-text-color, var(--text-color-inverse));
+      }
+
+      .dt-alert--primary:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--primary-color);
+        --dt-alert-context-background-color: var(--primary-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-light);
+      }
+
+      .dt-alert--alert:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--alert-color);
+        --dt-alert-context-background-color: var(--alert-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-light);
+      }
+
+      .dt-alert--caution:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--caution-color);
+        --dt-alert-context-background-color: var(--caution-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-dark);
+      }
+
+
+      .dt-alert--success:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--success-color);
+        --dt-alert-context-background-color: var(--success-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-light);
+      }
+
+
+      .dt-alert--inactive:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--inactive-color);
+        --dt-alert-context-background-color: var(--inactive-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-light);
+      }
+
+
+      .dt-alert--disabled:not(.dt-alert--outline) {
+        --dt-alert-context-border-color: var(--disabled-color);
+        --dt-alert-context-background-color: var(--disabled-color);
+        --dt-alert-context-text-color: var(--dt-alert-text-color-dark);
+      }
+
+
+      .dt-alert--primary.dt-alert--outline {
+        --dt-alert-context-border-color: var(--primary-color);
+        --dt-alert-context-text-color: var(--primary-color);
+      }
+
+      .dt-alert--alert.dt-alert--outline {
+        --dt-alert-context-border-color: var(--alert-color);
+        --dt-alert-context-text-color: var(--alert-color);
+      }
+
+      .dt-alert--caution.dt-alert--outline {
+        --dt-alert-context-border-color: var(--caution-color);
+        --dt-alert-context-text-color: var(--caution-color);
+      }
+
+
+      .dt-alert--success.dt-alert--outline {
+        --dt-alert-context-border-color: var(--success-color);
+        --dt-alert-context-text-color: var(--success-color);
+      }
+
+
+      .dt-alert--inactive.dt-alert--outline {
+        --dt-alert-context-border-color: var(--inactive-color);
+      }
+
+
+      .dt-alert--disabled.dt-alert--outline {
+        --dt-alert-context-border-color: var(--disabled-color);
+      }
+
+      button.toggle {
+        margin-inline-end: 0;
+        margin-inline-start: auto;
+        background: none;
+        border: none;
```

Comment:
If you add `color: (var(--dt-alert-context-text-color, var(--dt-alert-text-color)));` here it will fix the dismiss toggle color problem

Reply:
@micahmills, I solved it by letting the button inherit the alert text color.